.pylintrc

```diff
@@ -1,3 +1,7 @@
+[MASTER]
+# We did not write this script.  Changes should go upstream.
+ignore=build_manpage.py
```

Comment:
From where this script came from? I don't see it in any folder. I see it being imported, but not present on the project 🤔

Reply:
Ah yeah, this is from long enough ago that we were using this script but no longer.  So we can remove this portion of the change